src/snowflake/cli/api/entities/common.py

```diff
@@ -1,16 +1,20 @@
 from enum import Enum
+from typing import Generic, TypeVar
 
 
 class EntityActions(str, Enum):
     BUNDLE = "bundle"
 
 
-class EntityBase:
+TEntityModel = TypeVar("TEntityModel")
```

Comment:
nit: I think the python standard is to use single letters for type params, most typically `T`.

Reply:
Alright, changed to `T`, but personally I find it much less readable when it's not trivial to understand what T is